strings/capitalize.py

```diff
@@ -3,7 +3,13 @@
 
 def capitalize(sentence: str) -> str:
     """
-    This function will capitalize the first letter of a sentence or a word
+    Capitalizes the first letter of a sentence or word.
+
+    :param sentence: The input sentence or word to be capitalized.
+    :return: The sentence with the first letter capitalized.
+
+    Examples:
+
```

Comment:
```suggestion
```
This info doesn't need to be specified IMO because the description already makes this clear.